src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private String color;
```

Comment:
Field color is common for all figure, it makes sense to move this variable to Figure class, also type of this variable should be Color since you already have this enum